vaccine_feed_ingest/runners/ca/los_angeles_gov/normalize.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import logging
+import os
+import pathlib
+import re
+import sys
+from typing import List, Optional
+
+from vaccine_feed_ingest_schema import location as schema
+
+from vaccine_feed_ingest.utils.normalize import provider_id_from_name
+from vaccine_feed_ingest.utils.parse import location_id_from_name
+
+# Configure logger
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s %(levelname)s:%(name)s:%(message)s",
+    datefmt="%m/%d/%Y %H:%M:%S",
+)
+logger = logging.getLogger("ca/los_angeles_gov/normalize.py")
```

Comment:
(ok for followup)
There's now a shared logger! You can remove `import logging` at the top, and instead add (be sure to run `isort` after adding):

```suggestion
from vaccine_feed_ingest.utils.log import getLogger

logger = getLogger(__file__)
```